modules/caddyhttp/celmatcher.go

```diff
@@ -25,7 +25,7 @@ import (
 
 	"github.com/caddyserver/caddy/v2"
 	"github.com/caddyserver/caddy/v2/caddyconfig/caddyfile"
-	"github.com/gogo/protobuf/proto"
+	"github.com/golang/protobuf/proto"
 	"github.com/golang/protobuf/ptypes/timestamp"
```

Comment:
Note that the package github.com/golang/protobuf/proto is deprecated. We're instructed to use the "google.golang.org/protobuf/proto" package instead. I didn't want to change it now to avoid intrusive changes whose consequences are, currently, unknown.

Reply:
I wonder if we should, though, since deprecated packages spells lack of updates and fixes?

Reply:
Done. For a second I was worried this will break the code because cel-go uses the deprecated package `github.com/golang/protobuf/ptypes/timestamp`. I checked the code in `github.com/golang/protobuf/ptypes/timestamp` and found[ it uses type aliases](https://github.com/golang/protobuf/blob/d04d7b157bb510b1e0c10132224b616ac0e26b17/ptypes/timestamp/timestamp.pb.go#L15), so it will all work fine. Kinda cool the type alias works perfectly [as intended and argued for](https://talks.golang.org/2016/refactor.article).